### PR TITLE
bug_template: more specific information

### PR DESCRIPTION
**Pull request by matu3ba**
interaction with other programs, profile existence, lookup for installed profiles and user investigation on related problems

### CODE REVIEW

**Comment by matu3ba** [comment]
improvement of #3370 which hopefully reduces maintenance burden. 
Explicit asking for interaction with other programs and their existence in the profiles, so users may search or fix the places directly.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -2,36 +2,42 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: ''
+labels: 'review'
```
Comment (rusty-snake) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: There is no review label.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -2,36 +2,42 @@
...
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+**Bug and expected behavior**
+- Describe the bug.
+- What did you expect to happen?
 
-**Behavior change on disabling firejail**
-Describe clear and concise what changed calling `firejail --noprofile PROGRAM` in a shell.
+**Standard profile or disabling firejail**
```
Comment (rusty-snake) [inline .github/ISSUE_TEMPLATE/bug_report.md:15]:
```suggestion
**No profile or disabling firejail**
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (matu3ba) [reply .github/ISSUE_TEMPLATE/bug_report.md:5]: Intention was to relabel for sorting things after review to communicate the status of each issue ie information-needed, bug, feat, discussion, etc.  
Would you prefer another name or to remove?
For example #3345 needs more information, which is only visible when your check the issue.

Comment (rusty-snake) [reply .github/ISSUE_TEMPLATE/bug_report.md:5]: I would remove that for now, and move the label discussion to #2824.

**Comment by matu3ba** [comment]
@rusty-snake Do you want me to rebase with your changes or update here? Not sure if github bloats up the git log history.

**Comment by rusty-snake** [comment]
I can also choose `squash and merge`, so do what you want.

**Comment by matu3ba** [comment]
@rusty-snake any more concerns?

**Comment by rusty-snake** [comment]
Merged, Thanks.

**Pull request merged**